src/foam/u2/wizard/util/CircleIndicator.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard.util',
```

Comment:
Is this reall wizard specific? Would a more general package be better?

Reply:
This is me forgetting to move these models.

How is `foam.u2.tag.CircleIndicator`?